### DISCUSSION TITLE
Node with multiple handlers

### DISCUSSION THREAD

**Discussion by gitguru** [bcakmakoglu/vue-flow#1607] [Q&A] [answered]
I have a custom node with 4 handlers, top, left, bottom, right. Where top, left are targets and bottom, right are sources. But only top and bottom are used when diagram is drawn. 
What could it be?

**Comment by bcakmakoglu** [comment, marked as answer]
https://vueflow.dev/guide/handle.html#multiple-handles

> When using multiple handles of the same type (source or target), each handle needs to have a unique id.
